onebusaway-android/src/main/AndroidManifest.xml

```diff
@@ -35,6 +36,7 @@
     <uses-permission android:name="${applicationId}.permission.TRIP_SERVICE"/>
 
     <application
+            tools:replace="android:allowBackup"
```

Comment:
There was some reason why I haven't included `allowBackup` to date, although I don't recall off the top of my head.  Just writing a note so we can come back to this later.  I'm assuming this is needed by Embedded Social?